modules/output.nix

```diff
@@ -28,6 +28,17 @@ let
 in
 {
   options = {
+    env = mkOption {
+      type = with types; attrsOf str;
```

Comment:
Down the line we could expand the type to also accept e.g. `path` and `package`, but this is fine for an initial starting point.

Idk what type the NixOS or home-manager `environment` option uses?

Reply:
I thought environment variables could only be strings?

Reply:
nvm https://github.com/nixos/nixpkgs/blob/master/nixos/modules/config/shells-environment.nix#L41-L41

Reply:
I'm fine with keeping things simple in this PR and opening an issue regarding the other types we could support